frontend/src/components/Gym/GymStatus.tsx

```diff
@@ -33,6 +33,17 @@ interface Props {
   isOpen: boolean | null
 }
 
+/**
+ *
+ * @param isOpen (boolean) status of the gym
+ * @returns closed/open status of the gym
+ *
+ * @example
+ * Retrieves gym status from backend. If the gym is closed, display "Gym is Closed" in red
+ * <GymStatus isOpen={True} />
```

Comment:
another JSX syntax nitpicking haha, boolean `={true}` is redundant
```suggestion
 * <GymStatus isOpen />
```

Reply:
these are not wrong but just to standardise stylings around here abit; can refer to our style guides [here](https://www.notion.so/WIP-RHDevs-Frontend-Code-Style-Guide-b5f294c047f14f78b370031afb2c73c5) :)